boefjes/boefjes/plugins/kat_external_db/normalize.py

```diff
@@ -0,0 +1,71 @@
+import json
+import logging
+from ipaddress import IPv4Interface, ip_interface
+from typing import Iterator, Union
+
+from boefjes.job_models import NormalizerMeta
+from octopoes.models import OOI
+from octopoes.models.ooi.dns.zone import Hostname
+from octopoes.models.ooi.network import IPAddressV4, IPAddressV6, IPV4NetBlock, IPV6NetBlock, Network
+
+# Expects raw to be json containing a list of ip_addresses/netblocks
+# (as dictionaries) and a list of domains (as dictionaries).
+# The paths through the dictionaries (to the lists and through the lists)
+# are defined below.
+# T O D O add these variables as normalizer settings in UI.
+IP_ADDRESS_LIST_PATH = ["ip_addresses"]
+IP_ADDRESS_ITEM_PATH = ["ip_address"]
```

Comment:
Could these be defined as jsonpath queries for even more flexibility?

Reply:
Yes, these can be added later when better documentation is available for adding packages to normalizers